druid/examples/readme.md

```diff
@@ -138,18 +163,13 @@ cargo run --example split_demo
 An example of how to split a widget in 2 in various ways. This also includes having the user drag the border!! 
 
 ## Sub Window
+Not working, no sub-window seen?
```

Comment:
Are any of the issues part of this #1986?

Reply:
I'm on Linux so I don't think so, it also could be I don't know what a "sub window" is.. I was expecting some obvious, like a.new window to pop up. It would be great if you or some one who actually knows what to expect could try it!

Reply:
That example works correctly in Linux. I dont know the exact definition of sub window but think of tooltips. When you hover over ""Hello World! It's mine!, you'll see a tooltip. Or when you click button "Make sub window" it'll show a sub window you can drag around. Tooltips, new windows that your main app window spawns or a dropdown (not yet available in druid) are all sub windows.

Reply:
@sjoshid, I do see the "Tips! Are good" ToolTip when I hover over "Hello World! It's mine!".

I now see the "Drag me" sub_window. The problem is that the `set_position(Point::new(1000.0, 500.0))` causes
it to be "off screen" with my dual monitor setup. As you can see from the screenshot I have two monitors one horizontal and one vertical and for some reason the "Drag me" sub_window is not visible. But If I remove the call `set_position` then "Drag me" appears in the middle of the 400 x 400 main screen:

![image](https://user-images.githubusercontent.com/1024284/135135625-18353401-505e-4c32-8f96-50e5e9c42eb1.png)

So I'm also going to add that modification to this PR.